AISKU/src/Initialization.ts

```diff
@@ -463,7 +464,7 @@ export class Initialization implements IApplicationInsights {
     }
 
     public pollInternalLogs(): void {
-        this.core.pollInternalLogs();
+        this._internalLogPoller = this.core.pollInternalLogs();
```

Comment:
Lets not expose this as it will restrict what and how we might refactor this in the future.

Lets change to have a ```this.core.stopPollInternalLogs()```, lets also fix that if ```pollInternalLogs``` is called more than once it stops the previous generated interval.